### PR TITLE
changed panel desine to match site

### PR DESCRIPTION
**Pull request by dlangston**

### CODE REVIEW

File: ooiui/static/css/common/plotting.css [hunk 1]
```diff
@@ -134,6 +134,10 @@
     background: transparent;
   }
 
+  div#highcharts-0 {
+    width: auto!important;
```
Comment (birdage) [inline ooiui/static/css/common/plotting.css]: @dlangston remove important

File: ooiui/templates/science/plotting.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
                 </div>
                     <div class="row">
                         <div class="panel">
-                            <div class="panel-heading">
+                            <div class="panel-heading" style="padding-left:0px;">
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston should be in a css class

**Comment by dlangston** [comment]
@birdage

**Comment by dlangston** [comment]
8479 change wording of search result

- completed


8280 change meta data to metadata

- completed


in progress:

- working on the new popups for the map page

- need to add deployment information to assembly level


8377 note about plots being decimated

- 90% complete, need to adjust some stylings


- adding scrolling to the annotations and alerts and alarms page


need feedback:

-  https://uframe-cm.ooi.rutgers.edu/issues/4208


This e-mail message and any attached file is the property of the sender and is sent in confidence to the addressee only.
Internet communications are not secure and RPS is not responsible for their abuse by third parties, any alteration or corruption in transmission or for any loss or damage caused by a virus or by other means.
 
Any advice contained in this e-mail is for information purposes only.

RPS Group Plc, company number: 208 7786 (England). Registered office: 20 Western Avenue Milton Park Abingdon Oxfordshire OX14 4SH.
RPS Group Plc web link: <http://www.rpsgroup.com>

**Pull request merged**